### DISCUSSION TITLE
Is there a way to make an event occur daily?

### DISCUSSION THREAD

**Discussion by RedburnDrift** [TEHE-Studios/ExpandedHelicopterEvents#144] [Q&A]
Hey, just curious. I'm running a multiplayer server with a bit of players on it, and I'd like the events to be more common than usual (we're running 3 hour long days.) Is this possible to do with your mod? Also, using the debug tool it appears as though no events are scheduled 

![image](https://user-images.githubusercontent.com/128259319/226243715-fd8fb5a4-503e-417f-8a01-51c5b3051b26.png)

I've also tried manually spawning in events, but some of them do not work. (On my private testing server for example, the military emergency helicopter was able to be spawned manually, but a FEMA airdrop was not). Am I doing something wrong? 

![image](https://user-images.githubusercontent.com/128259319/226244077-ff5e4823-dfed-4632-a32b-cda629de38a3.png)

Using the same mods, in the same order, I cannot spawn these helicopters on the main server. Or, it says they spawn, but I cannot hear them :(. 

I am sorry for the possible inconvenience, I am very new to running a dedicated zomboid server. I love your mod idea and it works great when I can get it to. I just need to figure out what I'm doing wrong, haha.

**Comment by Chuckleberry-Finn** [comment]
If the events don't show up it may be that you or your friends are in the woods?

The system tries to make it a bit logical when targeting people. Also, if you've been active in one area but leave to another, events will target the area you were active - and depending on the path taken, you may not experience it.

If you want more events overall you can try to increase the sandbox options for frequency -- I'm not sure what impact longer days may have but it should still work. The scheduler rolls for an event every in-game hour.

**Reply by RedburnDrift** [reply]
Okay, thank you! I'll just turn them all up. If I set everything to insane, would that be near full chance of an event each day? Maybe more? Or less?

**Reply by Chuckleberry-Finn** [reply]
Depends on some circumstances, but you should be seeing a lot more for sure.
